### ISSUE TITLE
Debug hovers slow to dismiss

### ISSUE THREAD

**Issue by eamodio** [microsoft/vscode#107921]
Did we somewhat recently change the speed at which debug hovers are hidden? (And possibly shown?)

Here you can see the difference in time from the debug hovers and normal editor hovers

![debug-hovers](https://user-images.githubusercontent.com/641685/94857316-4f7bf680-03ff-11eb-9fa9-635507c59615.gif)

**Comment by eamodio** [comment]
I honestly feel that the normal hovers go away too quickly, and debug ones are too slow 😄

**Comment by isidorn** [comment]
We did not change this recently. We show the hover with a 300 ms delay, and hide it with a 600 ms delay.
Code pointer https://github.com/microsoft/vscode/blob/0ecb64a2c8945dd1193967019f0734af539ca9c3/src/vs/workbench/contrib/debug/browser/debugEditorContribution.ts#L42

I think we should just behave the same as our custom hovers. And we should just find the right number.
The editor hover is by default also 300ms to show. So I would not change that.

@alexr00 what do you think? What is the current speed of disimisal for the custom hover?
@eamodio suggestion on ms number is welcome

**Comment by alexr00** [comment]
For custom hovers, we show with a 1500ms delay for mac, and 500 delay for all other systems. We dismiss without any delay.

**Comment by isidorn** [comment]
@alexdima does the editor hover use the same dealy for showing as for hiding? The 300ms by default?

**Comment by alexdima** [comment]
@isidorn It is controlled by the option `editor.hover.delay`, which is 300 ms by default.

**Comment by isidorn** [comment]
So I will also just respect that setting. Both the show and the hide will respect that delay.
I think that makes most sense, especialy since the debug hover is in the editor.

**Comment by isidorn** [comment]
After trying it out and watching @hediet do a demo I decided that the debug hover is way too pesky
Usually it is a bit larger than the regular editor hover and can get in the way.

So I have increased the time to show to be the doulbe of `editor.hover.delay` - 600ms. Yeah it is a bit magically. But 1500ms on mac sounds a bit too much imho, and 500ms on all other sistems makes somewhat sense speed wise.
